website/src/pages/index.js

```diff
@@ -35,7 +35,7 @@ function HomepageHeader() {
         <p className="hero__subtitle">
           {siteConfig.tagline}
           <br/>
-          Also available as a <a href="https://eclipse-apoapsis.github.io/ort-server/">server</a>.
+          Also available as a <Link to="https://eclipse-apoapsis.github.io/ort-server/">server</Link>.
```

Comment:
Even if the docs are linked, a very brief motivation for the change would be nice to have in the commit message.

Reply:
Thanks, but doesn't the preloading only work for internal pages? At least that's how I read the docs.

Reply:
They just give some examples in the docs for the optimizations and they seem to apply to internal links, but maybe there is more to it, so I think it's just good practice to use `Link`.